GiniVision/Classes/GiniVision.swift

```diff
@@ -17,19 +17,19 @@ import UIKit
 @objc public protocol GiniVisionDelegate {
     
     /**
-     Called when the user has taken an image.
+     Called when the user has taken an image or picked a PDF.
      
-     - parameter imageData: JPEG image data including meta information.
+     - parameter fileData: JPEG image data including meta information or PDF data
      */
-    func didCapture(_ imageData: Data)
+    func didCapture(_ fileData: Data)
     
     /**
      Called when the user has reviewed the image and potentially rotated it to the correct orientation.
      
-     - parameter imageData: JPEG image data including eventually updated meta information.
+     - parameter fileData:  JPEG image data including eventually updated meta information or PDF Data
      - parameter changes:   Indicates whether `imageData` was altered.
     */
-    func didReview(_ imageData: Data, withChanges changes: Bool)
+    func didReview(_ fileData: Data, withChanges changes: Bool)
```

Comment:
Yes, I thought about this. But, where would you validate de document? Because when the file is imported with the "Open with..." option it does not go through CameraScreen.

Reply:
Yeah, that's true. I was thinking about this and realized, there is no reason to call this delegate method for pdfs since this serves to notify clients, that a review finished and if there were changes cancel the analysis and start it with the new Data. Pdfs cannot change so no need to call this. What do you think?

Reply:
Yes, you are right. Because it is only call when "Weiter" is pressed  👍